lgc/patch/Gfx9ConfigBuilder.cpp

```diff
@@ -1854,25 +1854,8 @@ void ConfigBuilder::buildPsRegConfig(ShaderStage shaderStage, T *pConfig) {
     depthExpFmt = EXP_FORMAT_32_R;
```

Comment:
I am not too familiar with this patch, but I think the intention is to set hasDepthExpFmtZero if none of sampleMask/fragStencilRef/fragDepth is used? If so, I created a (largely untested) fix in #853.